src/streamlink_cli/argparser.py

```diff
@@ -1064,6 +1074,14 @@ def build_parser():
         Example: "aac"
         """
     )
+    transport.add_argument(
+        "--ffmpeg-no-start-at-zero",
+        action="store_true",
```

Comment:
https://docs.python.org/2/library/argparse.html#action

```suggestion
        action="store_false",
```